sql-statements/sql-statement-drop-sequence.md

```diff
@@ -15,10 +15,6 @@ aliases: ['/docs-cn/dev/reference/sql/statements/drop-sequence/']
 
 ![DropSequenceStmt](/media/sqlgram/DropSequenceStmt.png)
 
-**OptTemporary:**
-
-![OptTemporary](/media/sqlgram/OptTemporary.png)
-
 **IfExists:**
 
 ![IfExists](/media/sqlgram/IfExists.png)
```

Comment:
This diagram is wrong. The empty branch is missing.

Reply:
Done